test/unit/plugin_test.rb

```diff
@@ -208,6 +220,25 @@ def test_register_allowed_template_helpers_and_variables
     Foreman::Renderer::ALLOWED_HELPERS.delete(:my_variable)
   end
 
+  def test_extending_rendering_helpers
+    refute Foreman::Renderer.public_instance_methods.include?(:my_helper)
+    refute_includes Foreman::Renderer::ALLOWED_HELPERS, :my_helper
+    refute ::TemplatesController.public_instance_methods.include?(:my_helper)
+
+    @klass.register :foo do
```

Comment:
what about making this line liners with `{}`? the empty `do end` block looks strange